precompiles/conviction-voting/ConvictionVoting.sol

```diff
@@ -80,6 +80,12 @@ interface ConvictionVoting {
     /// @param pollIndex Index of the poll
     function removeVote(uint32 pollIndex) external;
 
+    /// @dev Remove vote in poll with track
+    /// @custom:selector dd0988a6
+    /// @param pollIndex Index of the poll
+    /// @param trackId Id of the track
+    function removeSomeVote(uint32 pollIndex, uint16 trackId) external;
```

Comment:
maybe this is a better name
```suggestion
    function removeVoteForTrack(uint32 pollIndex, uint16 trackId) external;
```
because the difference between it and removeVote is only the additional trackId param

Reply:
I like this naming better. The other naming was confusing.